src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1241,4 +1242,11 @@ public static void onEntityEnterSection(Entity entity, long packedOldPos, long p
         MinecraftForge.EVENT_BUS.post(new EntityEvent.EnteringSection(entity, packedOldPos, packedNewPos));
     }
 
+    public static float getDamageBonus(Player player, ItemStack weapon, Entity target, float enchantmentBonus)
```

Comment:
Does the weapon need to be passed in? It's publicly available on the player.

Reply:
imo yes,
in the context of Mixins, calling `player#getMainHandItem` doesn't guarantee I get the same ItemStack that was used to get the initial damage bonus (e.g. a mod that allows dual wielding attacks could just pass in the offhand stack into `EnchantmentHelper#getDamageBonus`), so its better to have it as a explicit parameter that way anyone doing Mixins knows they have to supply the correct ItemStack to the event as well